infra-as-code/bicep/modules/vwanConnectivity/parameters/mc-vwanConnectivity.parameters.all.json

```diff
@@ -0,0 +1,113 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "parLocation": {
+      "value": "chinaeast2"
+    },
+    "parCompanyPrefix": {
+      "value": "alz"
+    },
+    "parVirtualHubAddressPrefix": {
+      "value": "10.100.0.0/23"
+    },
+    "parAzureFirewallTier": {
+      "value": "Standard"
+    },
+    "parVirtualHubEnabled": {
+      "value": true
+    },
+    "parVpnGatewayEnabled": {
+      "value": true
+    },
+    "parExpressRouteGatewayEnabled": {
+      "value": true
+    },
+    "parAzureFirewallEnabled": {
+      "value": true
+    },
+    "parNetworkDnsEnableProxy": {
+      "value": true
+    },
+    "parVirtualWanName": {
+      "value": "alz-vwan-chinaeast2"
+    },
+    "parVirtualHubName": {
+      "value": "alz-vhub-chinaeast2"
+    },
+    "parVpnGatewayName": {
+      "value": "alz-vpngw-chinaeast2"
+    },
+    "parExpressRouteGatewayName": {
+      "value": "alz-ergw-chinaeast2"
+    },
+    "parAzureFirewallName": {
+      "value": "alz-fw-chinaeast2"
+    },
+    "parAzureFirewallAvailabilityZones": {
+      "value": []
+    },
+    "parFirewallPoliciesName": {
+      "value": "alz-azfwpolicy-chinaeast2"
+    },
+    "parVpnGatewayScaleUnit": {
+      "value": 1
+    },
+    "parExpressRouteGatewayScaleUnit": {
+      "value": 1
+    },
+    "parDdosEnabled": {
+      "value": true
+    },
+    "parDdosPlanName": {
+      "value": "alz-ddos-plan"
+    },
+    "parPrivateDnsZonesEnabled": {
+      "value": true
```

Comment:
Mooncake does not support DDoS

Reply:
Just pushed a change for all instances of `parDdosEnabled` in mc- parameters files.